props/saber_fett263_buttons.h

```diff
@@ -5653,15 +5724,45 @@ SaberFett263Buttons() : PropBase() {}
           current_menu_angle_ = fusor.angle2();
           return true;
         }
-        if (menu_) MenuDialIncrement(1);
+        if (menu_) {
+	  MenuDialIncrement(1);
+#ifdef FETT263_SPECIAL_ABILITIES
+#if NUM_BUTTONS == 1
+        } else {
```

Comment:
I think the "} else {" should be befor the #if since it is both in the if and else branches.
(same below)

Reply:
Updated

Reply:
updated